### ISSUE TITLE
SpaceDock API now give game_version rather than ksp_version

### ISSUE THREAD

**Issue by Dazpoet** [KSP-CKAN/CKAN#1654]
This breaks our ability to index from spacedock and, to paraphrase, is less than ideal.

Example can be found [here](https://github.com/KSP-CKAN/NetKAN/pull/3560) which ran fine through Jenkins and was subsequently merged, however running the same file through netkan.exe now yields an error

```
648 [1] DEBUG CKAN.NetKAN.Transformers.SpacedockTransformer (null) - Writing ksp_version from SpaceDock:
649 [1] FATAL CKAN.NetKAN.Program (null) - Object reference not set to an instance of an object.
652 [1] FATAL CKAN.NetKAN.Program (null) -    at CKAN.NetKAN.Transformers.SpacedockTransformer.Transform(Metadata metadata)
   at CKAN.NetKAN.Transformers.NetkanTransformer.<Transform>m__0(Metadata transformedMetadata, ITransformer transformer)
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at CKAN.NetKAN.Transformers.NetkanTransformer.Transform(Metadata metadata)
   at CKAN.NetKAN.Program.Main(String[] args)
```

I tried poking about to figure out what needs to be changed and I think it's somewhere in [here](https://github.com/KSP-CKAN/CKAN/blob/master/Netkan/Sources/Spacedock/SDVersion.cs) but could be totally wrong :)

Marking this as critical and pinging @dbent @pjf @Olympic1 @politas @techman83 @Postremus who might be able to further locate and or fix the error.

**Comment by godarklight** [comment]
This was my fault - spacedock could add ksp_version as a field to that specific game if that's easier for you guys, although that probably wouldn't be ideal.

I shouldn't break API's... :-/

**Comment by Dazpoet** [comment]
Seems I might've been looking in the wrong place, looking at the [SD-transformer](https://github.com/KSP-CKAN/CKAN/blob/master/Netkan/Transformers/SpacedockTransformer.cs) I see a lot of mentions of `ksp_version` so maybe that's what needs to be changed?

**Comment by dbent** [comment]
Fixed in #1655
